ludwig/datasets/insurance_lite/config.yaml

```diff
@@ -0,0 +1,3 @@
+version: 1.0
+s3_download_urls: s3://datasets.us-west-2.predibase.com/insurance.csv
```

Comment:
Is this public? Maybe we can use the Kaggle API to pull the dataset.

Reply:
Fixed in the latest patch!